### PR TITLE
Enable EBS encryption by default

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Default configuration for now, no custom keys.

### CODE REVIEW

File: sevenseconds/config/ec2.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from ..helper import ActionOnExit, info
```
Comment (mikkeloscar) [inline sevenseconds/config/ec2.py]: `info` is unused.

Comment (aermakov-zalando) [reply sevenseconds/config/ec2.py]: Fixed.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**